ament_cmake_core/cmake/core/ament_execute_extensions.cmake

```diff
@@ -17,10 +17,17 @@
 #
 # :param extension_point: the name of the extension point
 # :type extension_point: string
+# :param EXCLUDE: List of packages that should be skipped
+# :type EXCLUDE: list of strings
 #
 # @public
 #
 macro(ament_execute_extensions extension_point)
+  cmake_parse_arguments(_ARG "" "" "EXCLUDE" ${ARGN})
+  if(_ARG_UNPARSED_ARGUMENTS)
+    message(FATAL_ERROR "ament_lint_auto_find_test_dependencies() called with "
```

Comment:
The error message seems to be copy-n-pasted from a different function without being updated.

Reply:
Fixed